### PR TITLE
[feat-5459]: Put search bar to top menu

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-5459](https://gemeente-amsterdam.atlassian.net/browse/SIG-5459)

- Put the search bar in the header on mobile
- Adjust styling according to design

note:
- should be properly tested on mobile

[SIG-5459]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5459?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by OscarBakker — CHANGES_REQUESTED** [review]

File: src/signals/IncidentMap/components/IncidentMap/styled.ts [hunk 1]
```diff
@@ -37,6 +40,39 @@ export const StyledMap = styled(Map)`
...
+  margin: ${themeSpacing(2, 0, 5, 0)};
   line-height: inherit;
 `
+
+export const WrapperMessage = styled.div<{ $isMobile: boolean }>`
+  ${($isMobile) => {
+    if ($isMobile)
+      return css`
+        position: absolute;
+        top: 124px;
```
[the remark this answers is not in the document]
Comment (vdegraaf) [reply src/signals/IncidentMap/components/IncidentMap/styled.ts:52]: it's not used. Removed it.

**Review by OscarBakker — APPROVED** [review]

File: src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx [hunk 2]
```diff
@@ -37,12 +41,11 @@ import {
 export const IncidentMap = () => {
   const [bbox, setBbox] = useState<Bbox | undefined>()
   const [map, setMap] = useState<MapType>()
-  const [mapMessage, setMapMessage] = useState<JSX.Element | string>('')
+  const [mapMessage, setMapMessage] = useState<JSX.Element | string>('wrvsfdg')
```
Comment (SireeKoolenWijkstra) [inline src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx:44]: I don't think this should be here.

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]
- [ ] When I look up an address then the styling of the search bar changes. The border is only half there. 
image

File: src/signals/IncidentMap/components/AddressLocation/AddressSearchMobile.tsx [hunk 3]
```diff
@@ -1,71 +1,61 @@
...
-  OptionsList,
-} from './styled'
+import { AddressSearchWrapper as Wrapper, OptionsList } from './styled'
 
 export interface Props {
   address?: string
   setCoordinates: (coordinates?: LatLngLiteral) => void
-  setShowAddressSearchMobile: (value: boolean) => void
+
+  onFocus: any
```
Comment (SireeKoolenWijkstra) [inline src/signals/IncidentMap/components/AddressLocation/AddressSearchMobile.tsx:16]: Shouldn't this be a function?

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]

**Review by SireeKoolenWijkstra — APPROVED** [review]
lgtm

**Pull request merged**